Stable package tests CI failure on Node.js v16

[CI](https://github.com/open-telemetry/opentelemetry-js/actions/workflows/unit-test.yml) is constantly failing on Node.js v16 for stable packages tests since https://github.com/open-telemetry/opentelemetry-js/commit/ee2342bdb03095ad771581b684a2b59e3bd3c905 was merged.

I tried to take a look local but for me even `npm install` fails with
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: opentelemetry@0.1.0
npm ERR! Found: webpack@4.46.0
npm ERR! node_modules/webpack
npm ERR!   webpack@"4.46.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^5.1.0" from terser-webpack-plugin@5.2.5
npm ERR! node_modules/terser-webpack-plugin
npm ERR!   terser-webpack-plugin@"5.2.5" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/packages/opentelemetry-sdk-trace-base/package.json
+++ b/packages/opentelemetry-sdk-trace-base/package.json
@@ -83,7 +83,7 @@
     "webpack": "4.46.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.2.0"
+    "@opentelemetry/api": ">=1.1.0 <1.2.0"
   },
   "dependencies": {
     "@opentelemetry/core": "1.0.1",

--- a/packages/opentelemetry-sdk-trace-node/package.json
+++ b/packages/opentelemetry-sdk-trace-node/package.json
@@ -59,7 +59,7 @@
     "typescript": "4.4.4"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.2.0"
+    "@opentelemetry/api": ">=1.1.0 <1.2.0"
   },
   "dependencies": {
     "@opentelemetry/context-async-hooks": "1.0.1",

--- a/packages/opentelemetry-sdk-trace-web/package.json
+++ b/packages/opentelemetry-sdk-trace-web/package.json
@@ -86,7 +86,7 @@
     "webpack-merge": "5.8.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.2.0"
+    "@opentelemetry/api": ">=1.1.0 <1.2.0"
   },
   "dependencies": {
     "@opentelemetry/core": "1.0.1",

